### ISSUE TITLE
"Invalid ANSI color sequence" warning on startup

### ISSUE THREAD

**Issue by kbd** [xonsh/xonsh#3243]
`xonfig` output:
```
$ xonfig
+------------------+-----------------+
| xonsh            | 0.9.9           |
| Python           | 3.7.4           |
| PLY              | 3.11            |
| have readline    | True            |
| prompt toolkit   | 2.0.9           |
| shell type       | prompt_toolkit2 |
| pygments         | 2.4.2           |
| on posix         | True            |
| on linux         | False           |
| on darwin        | True            |
| on windows       | False           |
| on cygwin        | False           |
| on msys2         | False           |
| is superuser     | False           |
| default encoding | utf-8           |
| xonsh encoding   | utf-8           |
| encoding errors  | surrogateescape |
+------------------+-----------------+
```

Giving xonsh another try, looks like there are still `LS_COLORS` issues. Here's what I get on startup:

```
$ xonsh
/usr/local/Cellar/xonsh/0.9.9/libexec/lib/python3.7/site-packages/xonsh/__amalgam__.py:7117: RuntimeWarning: Invalid ANSI color sequence "target", using "NO_COLOR" instead.
  warnings.warn(msg, RuntimeWarning)
```

I gave my LS_COLORS values [last time this came up](https://github.com/xonsh/xonsh/issues/3034). Please let me know if I can do anything to help diagnose. Thanks.

**Comment by gforsyth** [comment]
Hey @kbd thanks for reporting this!  And sorry for the delay in getting to it.

The warning that's firing looks a little weird to me.  Can you try commenting out your `LS_COLORS` and confirm that that is the source of the error?

The warning also seems to indicate that you have an escape code named `target` -- does that show up in any of your RC files?

**Comment by laloch** [comment]
@gforsyth, you can see it [here](https://github.com/trapd00r/LS_COLORS/blob/6fe816f95860e33523b79c2c768d229b546379b3/LS_COLORS#L44). Such a value is indeed valid according to `dircolors --print-database`:
```
...
LINK 01;36 # symbolic link. (If you set this to 'target' instead of a
           # numerical value, the color is as for the file pointed to.)
...
```

**Comment by scopatz** [comment]
Oh.  That is defintely not implemented.

**Comment by scopatz** [comment]
potential fix in #3255
